### ISSUE TITLE
Language of acceptance email

### ISSUE THREAD

**Issue by alex-cstl** [grokability/snipe-it#5554]
**Describe the bug**
Our school board is french and we must use french softwares for our users. So our Snipe-IT website is set to be in french ("Default Language" under the "Localization" menu).

By personal preferences, I decided to set my own personal profile to be in english. This should only affect ME and no one else.
But I noticed that if I check out an asset to a user, the check out email (acceptance) the user receives is written in ENGLISH, even though the main website is set to display in FRENCH and the user's profile is also in FRENCH. So, it seems Snipe-IT is relying on the language of MY profile (ENGLISH) to decide what language the email should be sent in, instead of the website's Default Language.

**To Reproduce**
1. Set Snipe-IT's Default Language to FRENCH (Options > Localization > Default Language > French)
2. Set your own user profile to ENGLISH (User's name > Edit Your Profile > Language > English, US)
3. Check out an asset (with acceptance enabled) to a user

The **receiving** user should receive an email and this email should be in the language of the **sending** user.

**Expected behavior**
I expect Snipe-IT to send check out emails (acceptance) to users using the MAIN and DEFAULT language set in the general preferences (under the Localization menu in the options), which is FRENCH in our case.

**Screenshots**
Not really applicable

**Server (please complete the following information):**
 - Snipe-IT Version: 4.3.0
 - OS: Windows Server 2012, R2
 - Web Server: IIS
 - PHP Version: 7.1.9

**Desktop (please complete the following information):**
 - OS: Windows 10, 1709
 - Browser: Chrome
 - Version: 66.0.3359.181

**Error Messages**
Not applicable

**Additional context**
- Is this a fresh install or an upgrade?: Upgrade from 4.1.11 to 4.3.0
- What OS and web server you're running Snipe-IT on: See "server" above

**Comment by snipe** [comment]
@alex-cstl That's a really good point. We didn't think through that use-case, so thanks for that! (It's interesting that it hasn't come up before, but I suppose most folks typically don't split languages across countries that way. 

So it seems to me, the emails should go out in the language of the recipient, UNLESS there is no value for the locale of that user, in which case it should be set to the locale in Settings > Localization.

**Comment by alex-cstl** [comment]
@snipe Indeed, the emails should go out in the language of the recipient. If there is no value, it should use the site's main and default language.
You're right, most users probably don't split languages! And if someone does, they probably don't check out items to other users. I am also not gonna check out many items to users, but still I might and I know other technicians on the road that will, and some of them also think like me: Native language of a software is the way to go for the best comprehension :)

But speaking of the "no value" case, does it really happen? The very first time I went into "Edit Your Profile", the language was already set to French, the same as the web site's default language.

Thanks for looking at the issue and have a great day!

**Comment by snipe** [comment]
![good-news](https://user-images.githubusercontent.com/197404/46168202-a49f3380-c24c-11e8-8a89-691ccbab514b.gif)

Looks like the newer version of Laravel will handle this more elegantly. (I had some janky, wacky nonsense staged for develop, but was hoping not to have to use it.)

https://github.com/laravel/framework/pull/25752

**Comment by alex-cstl** [comment]
Cool! Can't wait to test this out! We're still trailing in the update department... still using Snipe-IT 4.3.0...
I'll check with my tech (Hugo Raymond) if he'll be available to push the upgrade on our server.

As we speak, Hugo is trying to get the "LDAP Sync" working because right now, it crashes while attempting to read and import all of our users from our AD. He has a ticket on-going with your team in order to find a fix. A "work-around" has been found but it is not elegant and it is time-consuming (since we have to sync all of our OU one by one...).

Thanks!

**Comment by thumperest** [comment]
Dear All, we are encountring same issue. I arranged user's language as Romanian, and I am using Turkish. When I send acceptance mail, it sends a Turkish one. 
Version: v4.9.4 - build 4437 (master)

**Comment by umitsari** [comment]
Hello there i was used "Email List of All Assigned " buton to notification for users at Turkish Language. I  updated snipe it at  Version v6.2.3 - build 11936 (master) then the notification couldn't reach in turkish, only english. I tried the change localization settings and profile settings english to turkish but never changed received mails languages.

**Comment by skolarianer** [comment]
The exact same issue is still there (or back again) in 8.1.17. Admin uses English and users are using German. Mails to users are in English. If I switch admin's language to German, mails to users are in German, too.

**Comment by snipe** [comment]
@skolarianer is this happening on all emails, or only some?

**Comment by snipe** [comment]
(Also are you certain there is a complete translation for those emails? Everything will default to English if a translation string isn't present in the localized file.)

We haven't been hearing any other reports of this in some time, so this would be a regression if it's true. 

@bzeus can you test this please? 

- `APP_LOCALE` in the .env is in language A
- Site language in language B
- User language set to language C
- Admin sends an email (send all assigned, or another email), user should receive that in language C
- Email received should be in user's language (try to use one we know we have 90% or more translated)

**Comment by snipe** [comment]
@skolarianer @bzeus We'll also want to confirm whether this is via web action or command line scripts

**Comment by skolarianer** [comment]
It's via web action and I only tested it with acceptance mails including an EULA. Admin uses "English, UK" and the users "German informal". `mail.php` for "German informal" is at 93%, but all strings needed for the acceptance mail are present.

EDIT: "Require Signature" is enabled.

**Comment by snipe** [comment]
Can you try a few other emails as well to see if it's just that one notification or others?

**Comment by bzeus** [comment]
Checking now

**Comment by snipe** [comment]
@skolarianer - what version of Snipe-IT are you running?

**Comment by bzeus** [comment]
@snipe 
I can confirm the following so far:
When site language is set to "German" in this case, and Admin profile language is set to "English", and user language is set to "German", the following happens:
- Checkout assets, accessories, etc emails are sent in the Admin profile language, here English
- When sending "Email list of all assigned", either by the Admin or the user themselves, the email is sent in the user's language, here German
